node/src/chain_spec.rs

```diff
@@ -141,6 +141,14 @@ fn testnet_genesis(
 	endowed_accounts: Vec<AccountId>,
 	_enable_println: bool,
 ) -> GenesisConfig {
+	// base nft class for genesis block
+	let initial_state = vec![(
+		get_account_id_from_seed::<sr25519::Public>("Alice"),
```

Comment:
I can't tell that it matters which account creates the class, so I just used this one.